pulp_rpm/app/models/advisory.py

```diff
@@ -185,12 +185,12 @@ def to_createrepo_c(self, collections=[]):
         rec.pushcount = self.pushcount
 
         if not collections:
-            collections = self.collections.all()
+            collections = self.collections.all().order_by("name", "pulp_id")
```

Comment:
Can there be multiple collections with the same name?

Reply:
Yes - we have code in other places (advisory merge, for example) that has to deal with that already. name can also be empty :(.  All of this makes me sad, and adding pulp_id here makes me sadder. Willing to make changes here!